network/components/mux_rx.c

```diff
@@ -60,14 +60,28 @@ void rx_return(void)
             assert(!err);
 
             buffer.phys_or_offset = buffer.phys_or_offset - buffer_data_paddr;
-            microkit_arm_vspace_data_invalidate(buffer.phys_or_offset + buffer_data_vaddr, buffer.phys_or_offset + buffer_data_vaddr + buffer.len);
+            const uintptr_t buffer_vaddr = buffer.phys_or_offset + buffer_data_vaddr;
```

Comment:
sorry to nitpick, can you drop this `const` as well. It's mainly because it's not used anywhere else in the code.